CHANGES.rst

```diff
@@ -10,11 +10,15 @@ Unreleased
   Reddit's API.
 * Ability to submit image galleries with :meth:`.submit_gallery`.
 * Ability to specify modmail mute duration.
+* Ability to specify type of award, anonymity, and message when awarding a submission or
+  comment.
 
 **Changed**
 
 * :meth:`~.Subreddits.gold` is an alias of :meth:`~.Subreddits.premium`.
 * Drop support for Python 3.5, which is end-of-life on 2020-09-13.
+* :meth:`~.Submission.gild` is an alias of :meth:`~.Submission.award`.
```

Comment:
Instead of `changed` can we list this as, added. And then add gild as deprecated?

Reply:
Do you want the `Subreddits.gold` changed that way as well?

Reply:
Yes please.

Reply:
Okay go that changed as well.